src/06-loops.js

```diff
@@ -11,7 +11,14 @@
  *  //> 25
  *  (3 + 4 + 5 + 6 + 7 = 25)
  */
-function rangeSum(min, max) {}
+function rangeSum(min, max) {
+  let array = []
+  for(let i = min; i < max; i++){
+    array.push(i);
+  let reducer = (min, max) => min + max;
```

Comment:
Be careful getting help from online or elsewhere.  Could you talk me through every step of line 18 forward?  It's far more important that you know whats going on in code than to turn in something on time.  Here we have an ES6 function used as a callback to an ` array.reduce ` call - a pretty advanced higher order function.